src/Symfony/Bundle/FrameworkBundle/Command/AbstractConfigCommand.php

```diff
@@ -52,6 +52,44 @@ protected function listBundles(OutputInterface|StyleInterface $output)
         }
     }
 
+    protected function listNonBundleExtensions(OutputInterface|StyleInterface $output)
+    {
+        $title = 'Available registered extension aliases not from a bundle';
```

Comment:
(nitpick) non-bundle/bundle-less might be clearer

Reply:
Done